#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_fix_wrong_table_model_fpi_in_40.dpatch by Adam Di Carlo <aph@debian.org>
##     and Daniel Leidert (dale) <daniel.leidert@wgdd.de>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: * docbook-4.0/dbpoolx.mod: Fixes some 3.1.7 references, that should be
## DP:   4.0 FPIs.
## DP:
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-6>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>

@DPATCH@
diff -urNad docbook-xml-4.5~/docbook-4.0/dbpoolx.mod docbook-xml-4.5/docbook-4.0/dbpoolx.mod
--- docbook-xml-4.5~/docbook-4.0/dbpoolx.mod	2000-05-18 21:40:49.000000000 +0200
+++ docbook-xml-4.5/docbook-4.0/dbpoolx.mod	2007-03-25 22:48:52.000000000 +0200
@@ -3354,7 +3354,7 @@
 
 <!-- Reference SGML Open Exchange Table Model -->
 <!ENTITY % tablemodel 
-  PUBLIC "-//Norman Walsh//DTD CALS Table Model XML V3.1.7//EN" 
+  PUBLIC "-//Norman Walsh//DTD CALS Table Model XML V4.0//EN" 
   "calstblx.dtd">
 ]]>
 
@@ -3388,7 +3388,7 @@
 <!ENTITY % tbl.entry.mdl "(%para.char.mix; | %tabentry.mix;)*">
 
 <!ENTITY % tablemodel 
-  PUBLIC "-//Norman Walsh//DTD Exchange Table Model 19960430 XML V3.1.7//EN"
+  PUBLIC "-//Norman Walsh//DTD Exchange Table Model 19960430 XML V4.0//EN"
   "soextblx.dtd">
 ]]>
 
